README.md

```diff
@@ -1,9 +1,25 @@
 # Support
 
 Forms that create Zendesk tickets for requests coming from government agencies.
-The app is written in Rails and uses the zendesk_api gem to connect to Zendesk.
 
-### Getting started
+## Technical documentation
+
+This is a Ruby on Rails application which is not public facing. However, it does face GOV.UK employees and is used as a way to submit Zendesk tickets. 
+The Zendesk tickets are created either from the information submitted in the `support` app forms or via the information collected from public 
+users which is forwarded from the `feedback` app. The `feedback` app is public facing and serves its own contact forms.
```

Comment:
Maybe add a link to the feedback app?  It's not a dependency exactly, because it's really something that sits upstream of the support app, so it won't appear below in the dependencies section.